pkg/tsdb/prometheus/models/query_test.go

```diff
@@ -27,6 +27,8 @@ func TestParse(t *testing.T) {
 		queryJson := `{
 			"expr": "go_goroutines",
 			"refId": "A",
+			"interval": "1s",
+			"intervalMs": 1000,
```

Comment:
Are these changes still necessary?

Reply:
Let me check them once more. Probably we won't need them anymore. Good catch.